tests/e2e/env/utils/update-ready-page.js

```diff
@@ -25,11 +25,11 @@ const updateReadyPageStatus = async ( status ) => {
 		statusFilter = 'draft';
 	}
 
-	const getPostsResponse = await client.get(`${wpPagesEndpoint}?search=ready&status=${statusFilter}`);
-	if ( getPostsResponse.data && getPostsResponse.data.length > 0) {
+	const getPostsResponse = await client.get(`${wpPagesEndpoint}?search=ready&status=${statusFilter}` );
+	if ( getPostsResponse.data && getPostsResponse.data.length > 0 ) {
 		const pageId = getPostsResponse.data[0].id;
 		// Update the page to the new status
-		await client.post(`${wpPagesEndpoint}/${pageId}`, { 'status': status });
+		await client.put(`${wpPagesEndpoint}/${pageId}`, { 'status': status } );
```

Comment:
Curious on the change from the `post` to `put` here. The [WordPress API specifies using a `post` for the update](https://developer.wordpress.org/rest-api/reference/pages/#update-a-page), but it looks like both are supported (`put` definitely makes more sense for an update action in an API, so just trying to understand a bit more around the change, or if it's just for better REST API semantics).

Add spacing:

```suggestion
		await client.put( `${wpPagesEndpoint}/${pageId}`, { 'status': status } );
```

Reply:
> just trying to understand a bit more around the change, or if it's just for better REST API semantics

Mainly semantics. With the Woo REST API it has to be `PUT`. I think WP may have included support for `POST` due to some hosting providers not supporting `PUT` by default.

Reply:
Oh gotcha, thanks for the clarification!